src/Command/Index.php

```diff
@@ -138,6 +138,8 @@ protected function processIndex(IndexInterface $indexConfig): void
                 $oldIndex->removeAlias($indexConfig->getName());
                 $newIndex->addAlias($indexConfig->getName());
                 $oldIndex->flush();
+
+                $this->output->writeln(sprintf('<comment>-> "%s" index is active now</comment>', $newIndex->getName()));
```

Comment:
```suggestion
                $this->output->writeln(sprintf('<comment>-> %s is now active</comment>', $newIndex->getName()));
```

Reply:
Changed it like that, although I like it better with the quotation marks 🙈